src/anemoi/training/train/forecaster.py

```diff
@@ -140,10 +145,10 @@ def metrics_loss_scaling(config: DictConfig, data_indices: IndexCollection) -> t
             pressure_level.minimum,
         )
 
-        for key, idx in data_indices.model.output.name_to_index.items():
+        for key, idx in data_indices.internal_model.output.name_to_index.items():
             # Split pressure levels on "_" separator
             split = key.split("_")
-            if len(split) > 1:
+            if len(split) > 1 and split[1].isdigit():
```

Comment:
Let's change this to index -1 instead for multi-"_"-variables with pressure levels